packages/components/src/components/checkbox/checkbox.jsx

```diff
@@ -83,7 +84,11 @@ class Checkbox extends React.Component {
                 >
                     {!!this.state.checked && <Icon icon='IcCheckmark' color='active' />}
                 </span>
-                <span className={classNames('dc-checkbox__label', classNameLabel)}>{label}</span>
+                {(typeof this.props.renderlabel === 'function' && this.props.renderlabel(label)) || (
```

Comment:
```suggestion
                {(typeof this.props.renderLabel === 'function' && this.props.renderLabel(label)) || (
```

Reply:
render props should be lowercase, otherwise it raises an error.